### DISCUSSION TITLE
使用download进度下载，abort后出现报错

### DISCUSSION THREAD

**Discussion by chansee97** [alovajs/alova#309] [Q&A]
请求方法定义
![image](https://github.com/alovajs/alova/assets/90271994/0b503e1c-89b6-4b43-868b-9f6a118e246b)
请求方法调用
![image](https://github.com/alovajs/alova/assets/90271994/303d9113-709b-43b7-9cdb-9bdbb5f01391)
出现的控制台错误
![image](https://github.com/alovajs/alova/assets/90271994/6c756758-6c53-4683-969c-c52bf17e6c9f)

```
Uncaught (in promise) DOMException: BodyStreamBuffer was aborted
at Object.abort
```

abort中断后可以正常停止，但是控制台会出现这个错误。这个似乎有点问题
mdn关于abort的示例不会报错 https://mdn.github.io/dom-examples/abort-api/

**Comment by MeetinaXD** [comment]
问题已修复，请升级到2.20.0后解决。

https://github.com/alovajs/alova/issues/331
